chandra_aca/fov_plot.py

```diff
@@ -0,0 +1,309 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+from astropy.table import hstack, Table
+import agasc
+from Chandra.Time import DateTime
+import Quaternion
+from Ska.quatutil import radec2yagzag
+
+from .transform import pixels_to_yagzag
+
+# rc definitions
+frontcolor = 'black'
+backcolor = 'white'
+plt.rcParams['lines.color'] = frontcolor
+plt.rcParams['patch.edgecolor'] = frontcolor
+plt.rcParams['text.color'] = frontcolor
+plt.rcParams['axes.facecolor'] = backcolor
+plt.rcParams['axes.edgecolor'] = frontcolor
+plt.rcParams['axes.labelcolor'] = frontcolor
+plt.rcParams['xtick.color'] = frontcolor
+plt.rcParams['ytick.color'] = frontcolor
+plt.rcParams['grid.color'] = frontcolor
+plt.rcParams['figure.facecolor'] = backcolor
+plt.rcParams['figure.edgecolor'] = backcolor
+plt.rcParams['savefig.facecolor'] = backcolor
+plt.rcParams['savefig.edgecolor'] = backcolor
+
+
+BAD_STAR_COLOR = 'tomato'
+BAD_STAR_ALPHA = .75
+FAINT_STAR_COLOR = 'lightseagreen'
+FAINT_STAR_ALPHA = .75
+
+
+def symsize(mag):
+    # map mags to figsizes, defining
+    # mag 6 as 40 and mag 11 as 3
+    # interp should leave it at the bounding value outside
+    # the range
+    return np.interp(mag, [6.0, 11.0], [40.0, 3.0])
+
+
+def _plot_catalog_items(ax, catalog):
+    """
+    Plot catalog items (guide, acq, bot, mon, fid) in yang and zang on the supplied
+    axes object in place.
+
+    :param ax: matplotlib axes
+    :param catalog: data structure containing starcheck-style columns/attributes
+                    catalog records.  This can be anything that will work with
+                    astropy.table.Table(catalog).  A list of dicts is the convention.
+    """
+    cat = Table(catalog)
+    gui = cat[(cat['type'] == 'GUI') | (cat['type'] == 'BOT')]
+    acq = cat[(cat['type'] == 'ACQ') | (cat['type'] == 'BOT')]
+    fid = cat[cat['type'] == 'FID']
+    mon = cat[cat['type'] == 'MON']
+    for row in cat:
+        ax.annotate("%s" % row['idx'],
+                    xy=(row['yang'] - 120, row['zang'] + 60),
+                    color='red',
+                    fontsize=12)
+    ax.scatter(gui['yang'], gui['zang'],
+               facecolors='none',
+               edgecolors='green',
+               s=100)
+    for acq_star in acq:
+        acq_box = plt.Rectangle(
```

Comment:
Since the specific type of box is not important, don't give it a specifier.  Just call it a `box`.  That matches your convention in the next stanza.  When I something specific I tend to scan the code or wonder if it gets used later.